### PR TITLE
Update CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by mcalestino**
## Description
A few sentences describing the overall goals of the pull request's commits.

## Related Issues
List related issues.

## Testing
A few sentences describing what testing you've done, e.g., manual tests, automated tests, deployed in production, etc.

## Tasks That Must Be Done
- [ ] Did you update CHANGELOG.md?
- [ ] Did you add or update tests?
- [ ] Did you update documentation?
- [ ] Were there any special dev tricks you had to use to work on this code efficiently?
  + [ ] Did you add them to DEVELOPING.md?
- [ ] Is this a build change?
  + [ ] If so, did you test on both Mac and Linux?

## Other
* If this is a documentation change for a particular release, please open the pull request against the release branch.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -2,3 +2,14 @@
 
 Ambassador follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
 
+As contributors and maintainers of any datawire.io or Ambassador.io project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
```
Comment (kflynn) [inline CODE_OF_CONDUCT.md:5]: Maybe "the Ambassador project" is sufficient here -- this will be the CoC just for Ambassador, after all.

**Comment by bnfb** [comment]
@mcalestino Thanks for the PR but I hope you don't mind if I ask a question and push back a little: I believe in having a Code of Conduct but I'm curious why you are suggesting that the statement that the Ambassador project follows the CNCF Code of Conduct is insufficient? 

Unless we want the Ambassador Code of Conduct to be different than the CNCF Code of Conduct, it seems like referencing the CNCF Code of Conduct has a number of advantages including

1. being translated into many languages (useful for a world-wide open source project like Ambassador), and
2. evolving as the (large and diverse) CNCF community finds new and better ways to define acceptable/unacceptable conduct. 

So unless there is some reason for the Ambassador project to diverge from the CNCF Code of Conduct, it seems that we should just continue to make the strong statement that the Ambassador project follows the CNCF Code of Conduct. ...?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (mcalestino) [reply CODE_OF_CONDUCT.md:5]: Ok so just remove project and leave Ambassador? Should I remove Datawire as well?

**Comment by kflynn** [comment]
Having talked about this with @bnfb and @mcalestino, I think we're going to close this and stick with the link to the CNCF text -- it should simplify the upcoming donation of Ambassador to CNCF. Thanks @mcalestino!

**Pull request closed without merging**